test/test_manifest.json

```diff
@@ -2470,7 +2479,7 @@
     },
     {  "id": "issue4926",
        "file": "pdfs/issue4926.pdf",
-       "md5": "ed881c8ea2f9bc4be94ecb7f2b2c149b",
+       "md5": "e0c36aaee776eea59a3b0cdbe27bd27d",
```

Comment:
Even though this version of the PDF file seems to exhibit the same issue as the previous one, I'd still suggest updating the link to an older version instead:
`http://web.archive.org/web/20140802003728/http://www.conservation.ca.gov/cgs/information/publications/cgs_notes/note_17/Documents/note_17.pdf`.

My reasoning is that the old version is *half* the size of the new one, and it seems unnecessary to require contributors to download bigger files than strictly required.